blog/docs/articles/eventing-rabbitmq-announcement.md

```diff
@@ -0,0 +1,62 @@
+# Announcing Eventing RabbitMQ moving to GA
+
+**Author: Gab Satchi, Senior Software Engineer @ VMware**
+
+**Date: 2022-07-26**
+
+We’re excited to announce that [Eventing RabbitMQ](https://github.com/knative-sandbox/eventing-rabbitmq) is now GA (General Availability). With this milestone, the APIs provided will remain stable in future iterations.
+A quick start can be found [here](https://knative.dev/docs/eventing/broker/rabbitmq-broker/).
+
+# Key Features
+
+### Use any RabbitMQ instance
+
+Eventing RabbitMQ works well with RabbitMQ clusters provisioned using the [cluster operator](https://github.com/rabbitmq/cluster-operator) but can also be configured to use RabbitMQ instances provisioned by other means.
+Both broker and source have a `rabbitmqClusterReference` which can either reference a `RabbitMQCluster` created using the cluster operator or a `Secret` containing the credentials to a RabbitMQ instance.
+A full guide can be found [here](https://github.com/knative-sandbox/eventing-rabbitmq/tree/49787466f88b21fe022216a13c8321b334f92dc4/samples/external-cluster).
+
+### FIFO vs Throughput
+
+Eventing RabbitMQ, by default, operates in a first-in-first-out (FIFO) mode for events. However, there can be use cases where ordering isn't required and higher throughput may be preferred.
+Bother broker and source have a `parallelism` annotation (default 1) that can be configured to process more than 1 message in parallel. An example outlining both FIFO and high throughput modes can be found [here](https://github.com/knative-sandbox/eventing-rabbitmq/tree/49787466f88b21fe022216a13c8321b334f92dc4/samples/trigger-customizations).
+
+### Queue Type
+
+Quorum queues are a fault tolerant replicated queue type to be used in multi-node setups. The performance between quorum queues and classic queues are very close and Eventing RabbitMQ has opted to using quorum queues by default.
+The queue type remains configurable at the broker level and any triggers subscribed to a broker will inherit the queue type. Queue type can be configured using the `RabbitMQBrokerConfig` object:
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: RabbitmqBrokerConfig
+...
+spec:
+  queueType: "classic" #'classic | quorum'
+...
+```
+
+### Metrics
+
+Eventing RabbitMQ mirrors a subset of the eventing metrics found [here](https://knative.dev/docs/eventing/observability/metrics/eventing-metrics/). Due to the single-tenant nature, `Broker - Filter` metrics aren't applicable.
```

Comment:
In the repo it's titled as "RabbitMQ Eventing components", so I think a proper name for this needs to be decided on and used consistently. If the title isn't going to be updated in the sandbox repo, the docs should also refer to "RabbitMQ Eventing components" throughout, not "Eventing RabbitMQ"

Reply:
Updating the repo title and docs to have a consistent name for the project